waku/waku_test.go

```diff
@@ -1182,7 +1214,9 @@ func TestConfirmationReceived(t *testing.T) {
 		case err := <-errorc:
 			t.Log(err)
 		case <-time.After(time.Second * 5):
-			rw1.Close()
+			if err := rw1.Close(); err != nil {
+				t.Errorf("error closing MsgPipe, '%s'", err)
+			}
```

Comment:
[Error handling is good](https://github.com/status-im/status-go/pull/1957/files#r417328424)